src/it/projects/MSHADE-420/invoker.properties

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+# 
+#   http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+invoker.goals.1 = clean package -Duser.timezone=UTC -DfinalName=UTC
+invoker.goals.2 = package -Duser.timezone=Japan/Tokyo -DfinalName=Tokyo
```

Comment:
Attention, this will break in the future because you pass system properties as user properties. This must go into MAVEN_OPTS. @slawekjaranowski

Reply:
good catch, improvement done